ArduCopter/Copter.h

```diff
@@ -464,7 +464,7 @@ class Copter : public AP_Vehicle {
 #endif
 
     // Altitude
-    int32_t baro_alt;            // barometer altitude in cm above home
+    float baro_alt_m;                         // barometer altitude in cm above home
```

Comment:
let's fix the comment to the right

Reply:
I need to add "space cm space" to my searches.